packages/site/src/providers/AtlantisPreviewEditorProvider.tsx

```diff
@@ -250,6 +250,27 @@ const myTheme = EditorView.theme(
       borderRadius: "var(--radius-base)",
       padding: "var(--space-small)",
     },
+    ".ͼi": {
```

Comment:
This `not a c` character was making me question my sanity for a bit

Reply:
Note that with the exception of _maybe_ `interactive`, these are all semantically incorrect, ideally the mode-respecting palette that's defined in the `code-theme` file above could be applied to all of our code instances and consumed here